build.yml

```
@@ -1,6 +1,8 @@
 name: Build
 
-on: [push]
+on:
+  workflow_dispatch:
+  push:
```

Not sure if this is the correct syntax, but from looking at the [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-multiple-events), we should use:

```suggestion
on: [workflow_dispatch, push]
```

---

Both versions are correct (this was my patch, I just told Nika to open a PR for it). I used this syntax to be consistent with the `deploy-staging` and `deploy-production` workflows.

---

Ah, got it! Thanks 👍